apps/emqx_machine/src/emqx_machine_schema.erl

```diff
@@ -285,7 +285,7 @@ fields("node") ->
           #{}
          )}
     , {"etc_dir",
-       sc(string(),
+       sc(list_to_binary(string()),
```

Comment:
you can't do list_to_binary here as schema.

Reply:
the `string()` function is imported from `typerefl`
it actually means `typerefl:string()` which returns a tuple, but not a list.